packages/sources/coinmetrics-test/src/index.ts

```diff
@@ -1,12 +1,12 @@
 import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
-import { PriceAdapter } from '@chainlink/external-adapter-framework/adapter'
+import { Adapter } from '@chainlink/external-adapter-framework/adapter'
+import { priceRouter, lwbaRouter } from './endpoint'
 import { config } from './config'
-import { priceRouter } from './endpoint'
 
-export const adapter = new PriceAdapter({
+export const adapter = new Adapter({
```

Comment:
I think we should keep this as `PriceAdapter`. Only one endpoint needs to be a price endpoint which is the case.